iOS/Example/BeagleDemo/BeagleDemo/AppDelegate.swift

```diff
@@ -43,6 +43,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         dependencies.deepLinkHandler = deepLinkHandler
         dependencies.validatorProvider = validator
         dependencies.analytics = AnalyticsMock()
+        dependencies.logEnable = true
```

Comment:
according to [Swift API Guideline](https://swift.org/documentation/api-design-guidelines/#general-conventions) we should treat Boolean variables in a different way

![Screen Shot 2020-06-17 at 10 29 57](https://user-images.githubusercontent.com/6465618/84904648-ef4a8a00-b085-11ea-8792-019098fab874.png)

```suggestion
        dependencies.isLoggingEnabled = true
```